Prometheus install [ERROR]

### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

I have installation error of use Installation Prometheus Script,

```
-mv: cannot stat 'consoles/': No such file or directory
mv: cannot stat 'console_libraries/': No such file or directory

[ERROR] in line 32: exit code 0: while executing command mv consoles/ console_libraries/ /etc/prometheus/
```

### What settings are you currently utilizing?

Default Settings

### Which Linux distribution are you employing?

Debian 12

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

![image](https://github.com/user-attachments/assets/29076efe-e854-45f0-bbe0-1c6a9b14f696)


### Please provide detailed steps to reproduce the issue.

I have use the script for install Prometheus,
bash -c "$(wget -qLO - https://github.com/asylumexp/Proxmox/raw/main/ct/prometheus.sh)"

--- a/ct/prometheus.sh
+++ b/ct/prometheus.sh
@@ -66,7 +66,6 @@ if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_v
   tar -xf prometheus-${RELEASE}.linux-amd64.tar.gz
   cd prometheus-${RELEASE}.linux-amd64
   cp -rf prometheus promtool /usr/local/bin/
-  cp -rf consoles/ console_libraries/ /etc/prometheus/
   cd ~
   rm -rf prometheus-${RELEASE}.linux-amd64 prometheus-${RELEASE}.linux-amd64.tar.gz
   echo "${RELEASE}" >/opt/${APP}_version.txt

--- a/install/prometheus-install.sh
+++ b/install/prometheus-install.sh
@@ -47,8 +47,6 @@ Type=simple
 ExecStart=/usr/local/bin/prometheus \
     --config.file=/etc/prometheus/prometheus.yml \
     --storage.tsdb.path=/var/lib/prometheus/ \
-    --web.console.templates=/etc/prometheus/consoles \
-    --web.console.libraries=/etc/prometheus/console_libraries \
     --web.listen-address=0.0.0.0:9090
 
 [Install]

--- a/install/prometheus-install.sh
+++ b/install/prometheus-install.sh
@@ -29,7 +29,6 @@ wget -q https://github.com/prometheus/prometheus/releases/download/v${RELEASE}/p
 tar -xf prometheus-${RELEASE}.linux-arm64.tar.gz
 cd prometheus-${RELEASE}.linux-arm64
 mv prometheus promtool /usr/local/bin/
-mv consoles/ console_libraries/ /etc/prometheus/
 mv prometheus.yml /etc/prometheus/prometheus.yml
 echo "${RELEASE}" >/opt/${APPLICATION}_version.txt
 msg_ok "Installed Prometheus"

